transformer_engine/pytorch/module/base.py

```diff
@@ -1229,8 +1228,7 @@ def grad_output_preprocess(
             ):
                 grad_bias = grad_output.dequantize().view(-1, grad_output.shape[-1]).sum(dim=0)
             else:
-                # TODO(ksivaman): Re-add fusion once kernel is available.
-                if isinstance(quantizer, (Float8BlockQuantizer, NVFP4Quantizer)):
+                if isinstance(quantizer, Float8BlockQuantizer):
```

Comment:
With the changes from https://github.com/NVIDIA/TransformerEngine/pull/2007, we could probably remove this check entirely and blindly pass into `tex.bgrad_quantize`. A similar change could be made in `LayerNormMLP`.